app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1020,6 +1020,9 @@ def rbac_user_validate?
     if @edit[:new][:group].blank?
       add_flash(_("A User must be assigned to a Group"), :error)
       valid = false
+    elsif Rbac.filtered([MiqGroup.find_by_id(@edit[:new][:group])].compact).empty?
```

Comment:
The more strict we make it, the more specs we need to amend, the more conflicts happen with backports.